core/input/keyboard.js

```diff
@@ -281,6 +282,25 @@ Keyboard.prototype = {
         Log.Debug("<< Keyboard.allKeysUp");
     },
 
+    _checkAlt: function (e) {
```

Comment:
please leave a comment here about why this is necessary (I see you go into more detail below, but a brief

```js
// work around broken Alt key handling on FF for Windows (see below)
```

would be good here too